src/makeCalculator.js

```diff
@@ -37,7 +37,51 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: (num) => {
+      calculator.result += num;
+
+      return calculator;
+    },
+
+    subtract: (num) => {
+      calculator.result -= num;
+
+      return calculator;
+    },
+
+    multiply: (num) => {
+      calculator.result *= num;
+
+      return calculator;
+    },
+
+    divide: (num) => {
+      if (calculator.result === 0) {
+        calculator.result = 0;
+
+        return calculator;
+      }
+
+      calculator.result /= num;
```

Comment:
Not fixed. The division by 0 is still possible here

Reply:
but return 0. What can I do to fix it?